compiler/compile/OSRData.hpp

```diff
@@ -119,7 +119,7 @@ class TR_OSRCompilationData
                            int32_t symRefOrder, int32_t symSize, bool takesTwoSlots);
    void ensureSlotSharingInfoAt(const TR_ByteCodeInfo& T);
    void addInstruction(TR::Instruction* instr);
-   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo &bcInfo);
+   void addInstruction(int32_t instructionPC, TR_ByteCodeInfo bcInfo);
```

Comment:
Hey Andrew, sry the explanation was in the commit msg and I copied them to the comments of this PR. The intention of the loop in this API is just walking up the call stack but not change the bcInfo. Change reference type to normal type to prevent accidentally changing the argument value.